.github/workflows/release.yml

```diff
@@ -74,9 +92,24 @@ jobs:
           git tag v${{ steps.bump.outputs.version }}
           git push origin HEAD --tags
 
+      - name: Extract changelog description
+        id: changelog
+        run: |
+          python .github/scripts/get_latest_changelog.py > changelog.txt
```

Comment:
your script saves a changelog.txt and prints to stdout just a message. So this line should be just run the script, then read changelog.txt (making sure to read from .github/scripts/changelog.txt